### ISSUE TITLE
False positive when return errors.Newf()

### ISSUE THREAD

**Issue by m3dwards** [tomarrell/wrapcheck#43]
```
return nil, errors.Newf("Unknown node implementation: %v", n.Impl)
```

Gives me a false positive

**Comment by maxwedwards** [comment]
Using with golangci so the following fixes it for me:

.golangci file:
```yaml
linters-settings:
  wrapcheck:
    ignoreSigs:
      - .Errorf(
      - errors.New(
      - errors.Newf(
      - errors.Unwrap(
      - .Wrap(
      - .Wrapf(
      - .WithMessage(
      - .WithMessagef(
      - .WithStack(

```
